Kommander 2.1.1 Release Notes - rebased (#3986)
mesosphere/dcos-docs-site: pages/dkp/kommander/2.1/release-notes/2.1.1/index.md

---
layout: layout.pug
navigationTitle: Kommander 2.1.1 Release Notes
title: Kommander 2.1.1 Release Notes
menuWeight: 20
excerpt: View release-specific information for Kommander 2.1.1
enterprise: false
beta: false
---
**D2iQ&reg; Kommander&reg; version 2.1.0 was released on December 23, 2021.**

[button color="purple" href="https://support.d2iq.com/hc/en-us/articles/4409215222932-Product-Downloads"]Download Kommander[/button]

To get started with Kommander, [download](../../download/) and [install](../../install/) the latest version of Kommander.

<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>

## Release summary

This release provides new features and enhancements to improve the user experience, fix reported issues, integrate changes from previous releases, and maintain compatibility and support for other packages used in Kommander.

## New features and capabilities

### Ability to add catalog applications 

> With this release, you can now add catalog applications to your Kommander instances. The following applications are provided for you:

Reviewer: > The following applications are provided for you:

I think we need to mention that these 3 operators are workspace scoped?

Author: @cwyl02 - See change above - does this work?

> With this release, you can now add catalog applications to your Kommander instances. The following applications are provided for you:

Author: ```suggestion
With this release, you can now add catalog applications to your Kommander instances. Currently, we provide the following operator applications that are workspace-scoped:
```

Reviewer: Looks good to me!

> * Kafka
> * Spark
> * Zookeeper

Reviewer 2: ```suggestion
* [Kafka](../../workspaces/applications/catalog-applications/kafka-operator/)
* [Spark](../../workspaces/applications/catalog-applications/spark-operator/)
* [Zookeeper](../../workspaces/applications/catalog-applications/zookeeper-operator/)
```

Reviewer 2: Going to delete/resolve this - I see that link below, that'd be better for this whole topic.

Reviewer: And add Operator in the link text please

> * Kafka
> * Spark
> * Zookeeper

Author: ```suggestion
* Kafka operator
* Spark operator
* Zookeeper operator
```

Reviewer: Looks good, might also want to include @msdolbey 's changes to add links.

> For more information, see the catalog application documentation. 

Reviewer 3: @Colin1841 can you add a link?

> For more information, see the catalog application documentation. 

Reviewer 2: ```suggestion
For more information, see the [catalog application documentation](../../workspaces/applications/catalog-applications/).
```

Author: Thanks @msdolbey!

> For more information, see the catalog application documentation. 

Author: ```suggestion
For more information, see the [DKP application documentation](../../../../../../../workspaces/applications/catalog-applications/dkp-applications/ )
```

## Fixes and Improvements

> - Fixed an issue where the 'dkp delete' command could fail with a SIGSEGV when attempting to delete a DKP cluster from AWS where you have permanent credentials. (COPS-7109)

Reviewer 2: ```suggestion
- Fixed an issue where the `dkp delete` command could fail with a SIGSEGV when attempting to delete a DKP cluster from AWS where you have permanent credentials. (COPS-7109)
```

- Fixed an issue where the AWS `--region` or Azure `--location` installer flags were not being enforced in the target cluster. (COPS-7101)
- Corrected an issue where the `PreprovisionedInventory` object and SSH key secret were not moved to the target cluster when making the cluster self-managing.(COPS-7079)

> ## Component updates

Reviewer 3: are these the right component versions for Kommander 2.1.1?

Reviewer 3: Casey fixed these. thank you!

The following services and service components have been upgraded to the listed version:

- centralized-grafana: 18.1.1
- centralized-kubecost: 0.20.0
- cert-manager: 0.2.7

> - dex: 2.20.5

Reviewer 4: I see `2.9.10` for `dex` 🤔, maybe this is was meant to be the `external-dns` version below which is `2.20.5`
```suggestion
- dex: 2.9.10
```

Reviewer 3: hmm maybe we should list both?

Reviewer 3: yeah maybe someone overly deleted (could have been me idk)

- external-dns: 2.20.5
- fluent-bit: 0.16.2
- gatekeeper: 0.6.8
- grafana-logging: 6.16.4
- grafana-loki: 0.33.1
- istio: 1.9.1
- jaeger: 2.21.0
- karma: 2.0.0
- kiali: 1.29.1
- knative: 0.18.3
- kube-oidc-proxy: 0.2.5
- kube-prometheus-stack: 18.1.1
- kubecost: 0.20.0

> - kubefed: 0.8.1

Reviewer 4: ```suggestion
- kubefed: 0.9.0
```

- kubernetes-dashboard: 5.0.2

> - kubetunnel: 0.0.7

Reviewer 4: ```suggestion
- kubetunnel: 0.0.8
```

- logging-operator: 3.15.0
- metallb: 0.12.2
- minio-operator: 4.1.7
- nfs-server-provisioner: 0.6.0
- nvidia: 0.4.3

> - project-grafana-logging: 6.16.4

Reviewer 4: ```suggestion
- project-grafana-logging: 6.16.14
```

---
layout: layout.pug
navigationTitle: Kommander 2.1.1 Release Notes
title: Kommander 2.1.1 Release Notes
menuWeight: 20
excerpt: View release-specific information for Kommander 2.1.1
enterprise: false
beta: false
---
**D2iQ&reg; Kommander&reg; version 2.1.0 was released on December 23, 2021.**

[button color="purple" href="https://support.d2iq.com/hc/en-us/articles/4409215222932-Product-Downloads"]Download Kommander[/button]

To get started with Kommander, [download](../../download/) and [install](../../install/) the latest version of Kommander.

<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>

## Release summary

This release provides new features and enhancements to improve the user experience, fix reported issues, integrate changes from previous releases, and maintain compatibility and support for other packages used in Kommander.

## New features and capabilities

> With this release, we provide you the ability to deploy or delete the following Catalog Applications from Kommander:

Author: ```suggestion
With this release, we provide you the ability to deploy or delete the following Catalog Applications from the Kommander UI:
```

* Kafka
* Spark
* Zookeeper

> 

Author: @cbuto @cwyl02 - please add to & update this section as necessary. Is there anything beyond these 3 applications that we need to call out here?

Reviewer 4: I'm not sure if any other feature were added but this is what York and I worked on. Should we specify that these are operators? since the actual kafka/spark/zookeeper apps are manually deployed by users

Reviewer: I think we better use Kafka Operator, Spark Operator, Zookeeper Operator here.
